fpdf/fpdf.py

```diff
@@ -1976,7 +1938,9 @@ def cell(
         h=None,
         txt="",
         border=0,
-        ln=0,
+        new_x=XPos.RIGHT,
+        new_y=YPos.TOP,
+        ln=None,
```

Comment:
`ln='DEPRECATED'` would be more explicit and more consistent with how other deprecated arguments are currently handled